### DISCUSSION TITLE
wen Svelte?

### DISCUSSION THREAD

**Discussion by vseehausen** [WalletConnect/walletconnect-monorepo#4229] [Ideas]
How amazing would it be to have a svelte version of Web3modal?

**Comment by glitch-txs** [comment]
Hi Valle! I think we talked in the Wagmi's discord.

Web3Modal can be integrated already with Svelte, we plan to add first class support once Wagmi supports Svelte on a high level package.

You can check an example of how to integrate Web3Modal + Svelte here

https://github.com/WalletConnect/web-examples/tree/main/dapps%2Fweb3modal%2Fsvelte
